### DISCUSSION TITLE
How to begin migration from intel to m1?

### DISCUSSION THREAD

**Discussion by wang-jinbo** [Homebrew/discussions#3168] [Getting started]
I've used `homebrew` in intel for a long time, recently I'm trying to use migration assistant of 🍎 to migrate brew from my intel mac to m1 mac, which looks like to transfer the whole system, including a lot of intel-only binaries. What confuse me a lot is that there is no official migration guide, I read a lot of posts, Should I do:
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh)"
```
to remove old intel brew (`/usr/local/`) and then run
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
``` 
to install m1 native brew(`/opt/`), am I right? I hope this kind of instruction can be provided on official documentation, as fas as now, official doc seems not to provide a migration guide

**Comment by carlocab** [comment]
There is a thread about migration here: https://github.com/Homebrew/discussions/discussions/417

A contribution of a migration guide to the documentation would be much appreciated.

**Reply by wang-jinbo** [reply]
The problem is that I have plenty of universal apps and intel-only apps(no arm version available). Is it appropriate to move them forcely from `/usr/local/Caskroom` to `/opt/homebrew`? since these apps are too large.
